js/values/Paper.js

```diff
@@ -35,10 +45,26 @@ Paper.prototype.hasText = function() {
 
 /**
  * Return the associated text or am empty string if no text is available.
- * @returns {string}
+ * @returns {string} Returns text
  */
 Paper.prototype.getText = function() {
 	return this._text;
 };
 
+/**
+ * Check whether a metaDescription is available
+ * @returns {boolean} Returns true if metaDescription isn't empty
+ */
+Paper.prototype.hasMetaDescription = function() {
```

Comment:
Why would you need to check if metaDescription is available when getMetaDescription returns an empty string if there is none? It seems like this is duplicate now.